### ISSUE TITLE
Really needs a gif showing the TabBar in action

### ISSUE THREAD

**Issue by tciuro** [wxxsw/TabBar#2]
Developers would appreciate a screenshot or animated gif showing the control in action.

**Comment by wxxsw** [comment]
Yes, this project has stalled due to some technical limitations. But now tabItem {...} can be anything, not limited to Image and text, but there is no demo.
